file-browser/public/electron.js

```diff
@@ -9,14 +10,8 @@ function createWindow() {
     // Create the browser window.
     win = new BrowserWindow({width: 800, height: 600});
 
-    // and load the index.html of the app.
-    // from https://medium.freecodecamp.com/building-an-electron-application-with-create-react-app-97945861647c
-    const startUrl = process.env.ELECTRON_START_URL || url.format({
-            pathname: path.join(__dirname, 'build/index.html'),
-            protocol: 'file:',
-            slashes: true
-        });
-    win.loadURL(startUrl);
+    // load the url
+    win.loadURL(isDev ? 'http://localhost:3000' : `file://${path.join(__dirname, '../build/index.html')}`);
```

Comment:
This edit seems wrong. Can you double check?
- [ ] make the edit as minimal as possible.